internal/orchestrator/commiter.go

```diff
@@ -25,24 +24,23 @@ type Commiter struct {
 }
 
 func NewCommiter(storage storage.IStorage) *Commiter {
-	triggerInterval, err := strconv.Atoi(os.Getenv("COMMITER_TRIGGER_INTERVAL"))
-	if err != nil || triggerInterval == 0 {
+	committerConfig := config.GetCommitterConfig()
+	pollerConfig := config.GetPollerConfig()
```

Comment:
how about we load all configs as global variables at boot and use the vars everywhere without the need to import>get config>use

Reply:
Good idea, reduces code in the config. Changed